geoip.go

```diff
@@ -338,3 +338,19 @@ func (gi *GeoIP) GetCountry_v6(ip string) (cc string, netmask int) {
 	}
 	return
 }
+
+func (gi *GeoIP) GetDatabaseInfo() (databaseInfos string) {
```

Comment:
I was going to complain about the named return variable, but it looks like I was doing that elsewhere here too. :-)